src/osp/Resource/AssetImporter.cpp

```diff
@@ -122,7 +125,22 @@ void osp::AssetImporter::load_part(TinyGltfImporter& gltfImporter,
     pkg.add<PrototypePart>(gltfImporter.object3DName(id), std::move(part));
 }
 
-void osp::AssetImporter::load_sturdy(TinyGltfImporter& gltfImporter, Package& pkg)
+/* Explanation of resPrefix:
+   When a mesh is created in blender, the object itself has a name (the one that
+   shows up in the scene hierarchy), but the underlying mesh data within that
+   object actually has a unique name, usually the name of the primitive that was
+   used initially, unless it was changed. These names will be something like
+   "Cylinder.004" and are numbered to prevent name collisions within a blend
+   file. The issue is that multiple blend files can have a "Cylinder.004" and
+   unless the author renames the mesh object itself (which most people never
+   touch), when you export the scenes to glTF files and load them both here, you
+   end up with a resource key collision due to the repeated name. Passing
+   resPrefix around allows us to specify a unique prefix to prepend to the mesh
+   name (or any other resource that has the same problem) that is used
+   internally to avoid name conflicts.
+*/
```

Comment:
Not sure if this is the best place for this; I think it's worth having somewhere, and I put it near the implementation since this detail is mostly hidden from the user